python/py-cachetools/Portfile

```diff
@@ -0,0 +1,28 @@
+PortSystem          1.0
```

Comment:
please add a modeline as the first line ```# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4```, followed by a newline. Plaese see the [guide](https://guide.macports.org/#development.creating-portfile) for instructions.

Reply:
you missed the newline between the ```modeline``` and ```PortSystem``` ;)

Reply:
So I did; thanks!